utils/host/src/lib.rs

```diff
@@ -27,6 +28,7 @@ pub enum DAConfig {
     /// The default DA configuration.
     Default,
     /// The Celestia DA configuration.
+    #[cfg(feature = "celestia")]
```

Comment:
Why does this need to be gated? In the `config.rs` file, it expects `DAConfig` to be available in order to route to `args.da_config`. I don't think there's a need to gate this.